### ISSUE TITLE
some questions about the training steps

### ISSUE THREAD

**Issue by powermano** [endernewton/tf-faster-rcnn#259]
loss = layers['total_loss'] # Define the loss
    gvs = self.optimizer.compute_gradients(loss) # Compute the gradients with regard to the loss

In the paper of  Ren et al. they follow the alternating 4 steps training algorithm, however in your code,you calculate the total loss and optimize it. Does this operation improve or hurt the results?

thanks
